advanced-selenium/Mod1/1.06/src/test/java/com/saucelabs/advancedselenium/SauceLabsTest.java

```diff
@@ -1,4 +1,4 @@
-package test.java.com.saucelabs.advancedselenium;
+package com.saucelabs.advancedselenium;
```

Comment:
With the way the tests is set up, based on this: https://github.com/saucelabs-training/advanced-selenium/tree/new_base repo, the tests won't work unless they are named as they were before